packages/google_maps_flutter/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:typed_data';
 
+import 'package:flutter/material.dart';
```

Comment:
Analyzer warning:
>    info - test/fake_maps_controllers.dart:7:8 - The import of 'package:flutter/material.dart' is unnecessary because all of the used elements are also provided by the import of 'package:flutter/services.dart'. Try removing the import directive. - unnecessary_import

Reply:
Done

Reply:
Actually this causes an error on flutter stable

Reply:
Since `Color` is not exported in `package:flutter/services.dart` in flutter stable:

<img width="1017" alt="Screen Shot 2022-08-17 at 4 05 09 PM" src="https://user-images.githubusercontent.com/7896519/185232558-f7de18d2-1fa3-4451-921f-4387198d6a6a.png">